### PR TITLE
Branch test

### PR DESCRIPTION
**Pull request by Ambrish5211**

### CODE REVIEW

File: api/app.js [hunk 1]
```diff
@@ -1,4 +1,30 @@
...
+
+app.use("/api/auth", authRouter);
+
+ app.use("/home", (req, res) => {
+     res.status(200).json({
+         data: "all good",
+     });
+ });
+
+app.all('*',(req, res)=>{
```
Comment (rakshyak-98) [inline api/app.js:26]: we can take care for this at client level (front-end)

**Review by rakshyak-98 — CHANGES_REQUESTED** [review]

File: api/app.js (hunk 1 above)
Comment (Ambrish5211) [reply api/app.js:26]: ok bhaiya

File: api/config/db.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+const connectToDb = () => {
+ mongoose.connect(process.env.MONGO_URI)
+ .then((conn) => {
+  
+  console.log(`Connected to DB: ${conn.connection.host}`);
+ })
+ .catch((err) => {
+  console.log(err.message);
+  process.exit(1)
```
[the remark this answers is not in the document]
Comment (Ambrish5211) [reply api/config/db.js:12]: okk, will remove it

File: api/config/db.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+
+const connectToDb = () => {
+ mongoose.connect(process.env.MONGO_URI)
+ .then((conn) => {
+  
+  console.log(`Connected to DB: ${conn.connection.host}`);
+ })
+ .catch((err) => {
+  console.log(err.message);
```
[the remark this answers is not in the document]
Comment (Ambrish5211) [reply api/config/db.js:11]: ok bhaiya

**Pull request merged**